users/views.py

```diff
@@ -154,6 +155,15 @@ def user_create_confirm(request, token):
         user.confirm_token = ''
         user.is_active = True
         user.save()
+        user.circle_set.create(
+            name='Me',
+            is_editable=False
+        )
+        user.circle_set.create(
+            name='Public',
+            is_editable=False
+        )
```

Comment:
No because you cannot access them if you are not the owner of the circle, the relationship is created with the owner field, not with the members field